cli/cmdlineparser.cpp

```diff
@@ -1005,6 +1003,7 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
 
             else if (std::strncmp(argv[i], "--output-format=", 16) == 0) {
                 const std::string format = argv[i] + 16;
+                // TODO: text and plist is missing
```

Comment:
well.. plist requires a filepath option so you can't just use `--output-format=plist`. you still have to provide `--plist-output` also and then this is redundant.

And I didn't see the point to write `--output-format=text` that seems redundant also.

Reply:
> well.. plist requires a filepath option so you can't just use `--output-format=plist`. you still have to provide `--plist-output` also and then this is redundant.

Makes sense.

> And I didn't see the point to write `--output-format=text` that seems redundant also.

It would help to override some previously arguments which might be immutable.

Reply:
We still need some handling if `plist` is overriden since `--plist-output` sets two fields and `--output-format` only sets one of them. So the TODO is still valid.

Reply:
> We still need some handling if plist is overriden since --plist-output sets two fields and --output-format only sets one of them. So the TODO is still valid.

ok

Reply:
> We still need some handling if `plist` is overriden since `--plist-output` sets two fields and `--output-format` only sets one of them. So the TODO is still valid.

Posted fix for this as #7288.